src/blocks/DownloadBlock/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { fonts } from '@hedviginsurance/brand'
+import * as React from 'react'
+import styled from 'react-emotion'
+
+import {
+  ContentWrapper,
+  MOBILE_BP_DOWN,
+  SectionWrapper,
+} from '../../components/blockHelpers'
+import { BaseBlockProps } from '../BaseBlockProps'
+import { TABLET_BP_UP } from '../HeaderBlock/mobile'
```

Comment:
Should we keep these in one place? Most of them currently exist in blockHelpers. Do we want to move them into separate files that are more descriptive?

Reply:
woops nice catch! Haha i think i just imported it from the wrong place

Reply:
(ide magic is dangerous sometimes)